### PR TITLE
oy2 9916 wcv-ch

### PR DESCRIPTION
**Pull request by daniel-belcher**
added new measure

[ticket](https://qmacbis.atlassian.net/browse/OY2-9916?atlOrigin=eyJpIjoiYjljZjdkYmJmMGRjNGQ4MzhiN2M5YWFlNTlkMmZmYTEiLCJwIjoiaiJ9)

[instance](https://d1pf4rui5az5r9.cloudfront.net/)

#### Pull Request Creator Checklist

- [ ] This PR has an associated issue or issues.
- [ ] The associated issue(s) are linked above.
- [ ] This PR meets all acceptance criteria for those issues.
- [ ] This PR and linked issue(s) are adequately documented
- [ ] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.
- [ ] Someone has been assigned this PR.
- [ ] At least one person has been marked as reviewer on this PR.

#### Pull Request Reviewer/Assignee Checklist

- [ ] This PR has an associated issue or issues.
- [ ] The associated issue(s) are linked above.
- [ ] This PR meets all acceptance criteria for those issues.
- [ ] This PR and linked issue(s) are adequately documented
- [ ] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.

### CODE REVIEW

File: services/ui-src/src/measures/2021/WCVCH/data.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+export const data: DataDrivenTypes.PerformanceMeasure = {
+  questionText: [
+    "Percentage of children ages 3 to 21 who had at least one comprehensive well-care visit with a primary care practitioner (PCP) or an obstetrician/gynecologist (OB/GYN) during the measurement year.",
+  ],
+  questionListItems: [],
+  categories,
+  qualifiers,
+};
+// change to array
+export const dataSourceData: DataDrivenTypes.DataSource = {
```
Comment (abelzer212) [inline services/ui-src/src/measures/2021/WCVCH/data.ts:21]: This is the default data source so you don't need to include this in the data - Lines 21-47

**Review by abelzer212 — CHANGES_REQUESTED** [review]

**Comment by abelzer212** [comment]
Screen Shot 2022-03-29 at 1 59 18 PM
Screen Shot 2022-03-29 at 1 59 10 PM
Missing Validation: Ensure that “Total” NDR set is auto calculated from the according age ranges.
The calculated total rate should be 50, I manually entered 66 and it's not throwing an error

File: services/ui-src/src/measures/2021/WCVCH/validation.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      locationDictionary: GV.omsLocationDictionary(
+        OMSData(true),
+        PMD.qualifiers,
+        PMD.categories
+      ),
+      validationCallbacks: [
+        GV.validateDenominatorGreaterThanNumerator,
+        GV.validateRateNotZero,
+        GV.validateOMSTotalNDR,
+      ],
```
Comment (jphechter) [inline services/ui-src/src/measures/2021/WCVCH/validation.ts:42]: Do you need this one?
```suggestion
      GV.validateRateZero,
      ],
```

File: services/ui-src/src/measures/2021/WCVCH/validation.ts [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    ...GV.validateNumeratorsLessThanDenominators(
+      performanceMeasureArray,
+      OPM,
+      ageGroups
+    ),
+    ...GV.validateNoNonZeroNumOrDenom(performanceMeasureArray, OPM, ageGroups),
+    ...GV.validateRequiredRadioButtonForCombinedRates(data),
+    ...GV.ensureBothDatesCompletedInRange(dateRange),
+    ...GV.validateTotalNDR(performanceMeasureArray),
+  ];
```
Comment (jphechter) [inline services/ui-src/src/measures/2021/WCVCH/validation.ts:60]:
```suggestion
  GV.validateRateZero
  ];
```

Same question

**Pull request closed without merging**